utils/publish-legacy.sh

```diff
@@ -3,9 +3,12 @@ set -euo pipefail
 
 # Modifies the tagged packages to contain the legacy `now` name.
 # This file will be deleted on Jan 1, 2021.
+echo "Publishing legacy \"@now\" packages"
 
 __dirname="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
Let's also add a `echo __dirname` here and then also another echo on line 18 after `cd` so we can double check the directories.

Reply:
https://github.com/zeit/now/pull/4266/commits/12090571e494eb1ca9e3716620c17df3d81650ef